docs/ref/django-admin.txt

```diff
@@ -793,6 +793,17 @@ be warned that using ``--fake`` runs the risk of putting the migration state
 table into a state where manual recovery will be needed to make migrations
 run correctly.
 
+.. django-admin-option:: --plan
+
```

Comment:
Add `.. versionadded:: 2.2` annotation and a release note in `docs/releases/2.2.txt`.

Reply:
Will do.